src/styles/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 100%;
+
+}
+
+.nav {
+  margin-left: 50px;
+
```

Comment:
Per BEM:

1 Each block should be in a separate file
2 No margins on BEM blocks